gradle.properties

```diff
@@ -46,4 +48,4 @@ kotlinLoggingVersion=2.1.21
 slf4jVersion=1.7.36
 
 # crt
-crtKotlinVersion=0.6.4
\ No newline at end of file
+crtKotlinVersion=0.6.5-SNAPSHOT
```

Comment:
NOTE: This will affect the next release and require releasing crt kotlin first and updating the version here, we need some automation to check for SNAPSHOT versions during release probably...